src/execution/vm/bytecode_generator.cpp

```diff
@@ -442,6 +442,55 @@ void BytecodeGenerator::VisitReturnStmt(ast::ReturnStmt *node) {
 void BytecodeGenerator::VisitSqlConversionCall(ast::CallExpr *call, ast::Builtin builtin) {
   ast::Context *ctx = call->GetType()->GetContext();
   switch (builtin) {
+    case ast::Builtin::IsNull: {
```

Comment:
These aren't really conversion calls. Can you pull it into a separate method?

Reply:
Pulled out.